tests/views/test_response_correlation.py

```diff
@@ -48,3 +57,54 @@ def test_response_selector_sorting(mock_data, dash_duo):
     assert response_list[1] == "test_response_4"
     assert response_list[2] == "test_response_44"
     assert response_list[3] == "test_response_99"
+
+
+def test_axes_labels(mock_data, dash_duo):
```

Comment:
they don't quite - I fixed `Comparison` first, and later (while reviewing my changes) noticed that `Correlation` was not fixed! Each Controller required fixing. I guess that should maybe be refactored...

Reply:
it could be refactored into a function that get's a loaded ensemble, a response (name), and a figure, and has the side effect of applying axes labels to the figure based on the index of the data of the given ensemble and response, and said function could be used in both controllers - but we'll leave it as is now.